tests/remote/remote_controller.pm

```diff
@@ -49,7 +50,13 @@ sub run {
     ensure_serialdev_permissions;
 
     if (check_var("REMOTE_CONTROLLER", "vnc")) {
-        select_console 'x11';
+        if(is_opensuse()) {
+            select_console 'x11';
+        } else
+        {
+            select_console 'root-console';
+            send_key('alt-f7', 10);
```

Comment:
`send_key` should not be called with a second positional argument. The format is

```
send_key($key [, wait_screen_change => $wait_screen_change])
```
with `$wait_screen_change` being a boolean value, if this is what you wanted to achieve.

Reply:
Hmm, this is old code which was there before, seems that parameter was ignored. Nice catch!